### PR TITLE
Added 1.20 Support

### PR DESCRIPTION
**Pull request by ImYenil**
Added 1.20 Support

### CODE REVIEW

**Comment by KKiiya** [comment]
Shouldnt we wait for minecraft to update 1.20? Like, they probablly release 1.20.1 or smth like that

**Comment by ImYenil** [comment]
> Shouldnt we wait for minecraft to update 1.20? Like, they probablly release 1.20.1 or smth like that

Since we don't know when version 1.20.1 will be released, we will add version 1.20 and drop support for version 1.20 when version 1.20.1 is released

**Comment by Xxyuri2005xX** [comment]
cannot be accepted until TAB puts support for 1.20 and later

**Comment by Xxyuri2005xX** [comment]
I correct myself, TAB V4 supports 1.20

File: versionsupport_v1_20_R1/.gitignore
Comment (tomkeuper) [inline versionsupport_v1_20_R1/.gitignore:1]: Quite sure version support modules dont have their own gitignore file

**Comment by tomkeuper** [comment]
Can you tell me what the differences are compared to 1.19 and what stuff has been changed?

**Comment by ImYenil** [comment]
> Can you tell me what the differences are compared to 1.19 and what stuff has been changed?

Just added support for version 1.20. This is because nms always changes as the version is updated.

**Comment by tomkeuper** [comment]
> > Can you tell me what the differences are compared to 1.19 and what stuff has been changed?
> 
> Just added support for version 1.20. This is because nms always changes as the version is updated.

and im asking you what changed

**Comment by ImYenil** [comment]
> > > Can you tell me what the differences are compared to 1.19 and what stuff has been changed?
> > 
> > 
> > Just added support for version 1.20. This is because nms always changes as the version is updated.
> 
> and im asking you what changed

What do you want? I just added new version support. I don't understand why more information is needed.

**Comment by tomkeuper** [comment]
> What do you want? I just added new version support. I don't understand why more information is needed.

Because you added the new version it needs to be clear what the differences are. If something breaks then we know where to look. And since Minecraft changes a bunch of stuff each update we need to say what we changed.

**Comment by Xxyuri2005xX** [comment]
this pull wanting could be closed and use the 1.20 support of andrei that I have seen and it works

**Comment by tomkeuper** [comment]
There are also a few things wrong so I will try to do this today

**Pull request closed without merging**